nodes/celestia-app.md

```diff
@@ -13,8 +13,45 @@ import mochaVersions from '/.vitepress/constants/mocha_versions.js'
 import coffeeVersions from '/.vitepress/constants/coffee_versions.js'
 </script>
 
-This tutorial will guide you through installing celestia-app. This
-tutorial presumes you completed the steps in
+This tutorial will guide you through installing celestia-app, both with
+[a pre-built binary](#installing-a-pre-built-binaries) and
+[from source](#building-binary-from-source).
+
+## Installing a pre-built binaries
+
+Installing a pre-built binary is the fastest way to get started with your
+Celestia consensus node. Versions containing
+[PR 2654](https://github.com/celestiaorg/celestia-app/pull/2654)
+will have these binaries available.
```

Comment:
[optional] we could replace this with something like:

```suggestion
Celestia consensus node. Releases after celestia-app v1.3.0 should have these binaries available.
```

so users don't have to track down which releases contain that PR. As you've already notices v1.2.0 doesn't contain the pre-built binaries because CI failed on that release and re-running CI won't resolve b/c the links remain broken in the v1.2.0 codebase.

Reply:
i like it. even if using `git tag/branch --contains` it is still very clunky UX